index.html

```diff
@@ -342,9 +346,9 @@ <h3><a class="textdecoration font1" href="#">Uber Navigation</a></h3>
                 <ul class="ul1">
 
                     <li><a class="textdecoration font2" href="#">CANOPY</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
                     <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
-                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
```

Comment:
- [ ] Kindly make sure to the alternative text `alt` to describe your image as required.